src/Fist/Http/src/Request.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace Fist\Http;
+
+class Request
```

Comment:
__[phpmd]__
* The class Request has 11 public methods. Consider refactoring Request to keep number of public methods under 10.
* The class Request has an overall complexity of 51 which is very high. The configured complexity threshold is 50.

<!-- Comment by [SideCI](https://sideci.com) -->